xbmc/cores/VideoPlayer/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 set(SOURCES DVDAudio.cpp
             DVDClock.cpp
+            VideoReferenceClock.cpp
```

Comment:
the list should be kept sorted, so move this at the end. same for HEADERS below.